web-app/django/VIM/apps/instruments/views/instrument_list.py

```diff
@@ -261,14 +281,36 @@ def _get_solr_page_results(
             **query_params,
             "rows": page_size,
             "start": start,
+            "hl": "true",
+            "hl.fl": "text",
+            "hl.simple.pre": "<b>",
+            "hl.simple.post": "</b>",
+            "hl.snippets": 1000,
         }
         # Remove our custom params
         lang_code = solr_params.pop("lang_code")
 
         solr_response = solr.search(**solr_params)
-        instruments = [
-            SolrInstrument(doc, lang_code=lang_code) for doc in solr_response.docs
-        ]
+
+        # Extract highlight info
+        highlights = getattr(solr_response, "highlighting", {})
```

Comment:
I personally prefer to rename `highlights` to `highlight_info` or `highlight_data`.